medusa/providers/torrent/torznab/torznab.py

```diff
@@ -179,7 +179,7 @@ def parse(self, data, mode):
                     peers_attr = item.find('torznab:attr', attrs={'name': 'peers'})
                     seeders = int(seeders_attr.get('value', 0)) if seeders_attr else 1
                     peers = int(peers_attr.get('value', 0)) if peers_attr else 0
-                    leechers = peers-seeders if peers-seeders>0 else 0
+                    leechers = peers - seeders if peers - seeders>0 else 0
```

Comment:
Also the >

Reply:
Already added spaces to the >, you're commenting on an old commit